src/kadmin/ktutil/ktutil.c

```diff
@@ -149,6 +150,7 @@ void ktutil_add_entry(argc, argv)
         }
         if ((strlen(argv[i]) == 2) && !strncmp(argv[i], "-k", 2)) {
             kvno = (krb5_kvno) atoi(argv[++i]);
+            use_kvno++;
```

Comment:
@greghudson The requested changes have been made. Let me know if there is anything else I can do.